### PR TITLE
Add acceuille and forgot password

### PR DESCRIPTION
**Pull request by am45880**
comment

### CODE REVIEW

File: src/main/java/com/arachidsocial/controllers/pages/AccueillePage.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package com.arachidsocial.controllers.pages;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+
+/**
+ * Created by Yassine on 2017-05-23.
+ */
+@Controller
+public class AccueillePage {
```
Comment (banctilrobitaille) [inline src/main/java/com/arachidsocial/controllers/pages/AccueillePage.java]: Typo : Accueil

File: src/main/java/com/arachidsocial/controllers/pages/AccueillePage.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+    @GetMapping("/accueille")
```
Comment (banctilrobitaille) [inline src/main/java/com/arachidsocial/controllers/pages/AccueillePage.java]: same :point_up:

File: src/main/java/com/arachidsocial/controllers/pages/AccueillePage.java [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    public String loadPage(){
+        return "accueille";
```
Comment (banctilrobitaille) [inline src/main/java/com/arachidsocial/controllers/pages/AccueillePage.java]: Same

File: src/main/java/com/arachidsocial/controllers/pages/forgotPassword.java [hunk 4]
```diff
@@ -0,0 +1,16 @@
+package com.arachidsocial.controllers.pages;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+
+/**
+ * Created by Yassine on 2017-05-23.
+ */
+@Controller
+public class forgotPassword {
```
Comment (banctilrobitaille) [inline src/main/java/com/arachidsocial/controllers/pages/forgotPassword.java]: Class name not CamelCase

File: src/main/java/com/arachidsocial/controllers/pages/forgotPassword.java [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+
+    @GetMapping("/forgot_password")
+    public String loadPage(){
+        return "forgot-password";
```
Comment (banctilrobitaille) [inline src/main/java/com/arachidsocial/controllers/pages/forgotPassword.java]: Template not following standard template_name

File: src/main/resources/templates/accueille.html [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+    <th:block layout:fragment="js-import">
+
+    </th:block>
+
+</head>
+<body>
+
+    <div class="col-md-8 content-holder" layout:fragment="content" >
+
+            <div id="myCarousel" class="carousel slide" data-ride="carousel" >
```
Comment (banctilrobitaille) [inline src/main/resources/templates/accueille.html]: myCarousel ??? should be more explicit

File: src/main/resources/templates/accueille.html [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+
+
+
+                <div class="carousel-inner">
+
+                    <div class="item active" >
+                        <img src="..\images\accueille-slider\event.png" style="vertical-align: middle;"  />
```
Comment (banctilrobitaille) [inline src/main/resources/templates/accueille.html]: typo Accueil

File: src/main/resources/templates/forgot-password.html [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+<body>
+
+    <div class="col-md-8 content-holder" layout:fragment="content" >
+        <div class="modal-body">
+            <div class="container-fluid">
+                <form class="form-horizontal" role="form">
+                    <h4>Forgot Password</h4>
+                    <hr/>
+                    <div class="form-group">
+                        <label for="newUsername">Username</label> <br/>
```
Comment (banctilrobitaille) [inline src/main/resources/templates/forgot-password.html]: Its not a new username

**Comment by alexandremalo** [comment]
BUMP

**Pull request closed without merging**